src/applications/appeals/996/tests/validations/validations.unit.spec.js

```diff
@@ -146,4 +146,12 @@ describe('contactInfoValidation', () => {
     );
     expect(addError.called).to.be.false;
   });
+  it('should not throw an error when addError function is missing', () => {
+    try {
+      contactInfoValidation();
+      expect(true).to.be.true;
+    } catch (error) {
+      expect(error).to.be.null;
```

Comment:
So the way that `try catch` works, is `catch` like the `else` in an `if else` statement where so long as `try` isn't true, then whatever happens is considered the "error"?

Reply:
See comment above 😸